MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -79,18 +79,18 @@ MeshLib::NodePartitionedMesh* NodePartitionedMeshReader::read(
 
     if(!BaseLib::IsFileExisting(fname_new)) // doesn't exist binary file.
     {
-        INFO("-->Reading ASCII mesh file ...");
+        INFO("Reading ASCII mesh file ...");
 
         mesh = readASCII(file_name_base);
     }
     else
     {
-        INFO("-->Reading binary mesh file ...");
+        INFO("Reading binary mesh file ...");
 
         mesh = readBinary(file_name_base);
     }
 
-    INFO("\t\n>>Total elapsed time in reading mesh:%f s\n", timer.elapsed());
+    INFO("Total elapsed time in reading mesh: %f s.", timer.elapsed());
```

Comment:
Other outputs of time measurements are formatted as `INFO("[time] ...`. I think this occurence here should be formatted the same way. Afterwards 👍